packages/shared/routes/dashboard/wallet/views/Send.svelte

```diff
@@ -405,7 +405,7 @@
     // TODO addlabel
```

Comment:
We can ignore this file, need to delete it just need to double check we have covered the logic in new flows